script.js

```diff
@@ -0,0 +1,432 @@
+
+    var checkIfSpell = true;
+    var classCheck = "";
+    //get all the inputs from the localstorage
+    if (localStorage.length != 0) {
+        //hit point local storage
+        if (localStorage.getItem("hitPoint") != "undefined") {
+            var hitPoint = localStorage.getItem("hitPoint", hitPoint);
+            $("#hp").val(hitPoint);
+
+        }
+        //armorclass local storage
+        if (localStorage.getItem("armorClass") != "undefined") {
+            var armorClass = localStorage.getItem("armorClass", armorClass);
+            $("#ac").val(armorClass);
+        }
+        //speed local storage
+        if (localStorage.getItem("speed") != "undefined") {
+            var speed = localStorage.getItem("speed", speed);
+            $("#sp").val(speed);
+        }
+        //initiative local storage
+        if (localStorage.getItem("initiative") != "undefined") {
+            var initiative = localStorage.getItem("initiative", initiative);
+            $("#int").val(initiative);
+        }
+        //profeciency local storage
+        if (localStorage.getItem("prof") != "undefined") {
+            var prof = localStorage.getItem("prof", prof);
+            $("#pb").val(prof);
+        }
+        //character local storage
+        if (localStorage.getItem("character") != "undefined") {
+            var character = localStorage.getItem("character", character);
+            $("#char-name").val(character);
+        }
+        //class local storage
+        if (localStorage.getItem("classbtn") != "Classarrow_drop_down") {
+            var classbtn = localStorage.getItem("classbtn", $('.classbtn').text());
+            $('.classbtn').text(classbtn);
+            classCheck = $('.classbtn').text();
+            fillSpell();
+
+        }
+        //level local storage
+        if (localStorage.getItem("levelbtn") != "Levelarrow_drop_down") {
+            var levelbtn = localStorage.getItem("levelbtn", $('.levelbtn').text());
+            $('.levelbtn').text(levelbtn);
+        }
+        //race local storage
+        if (localStorage.getItem("racebtn") != "Racearrow_drop_down") {
+            var racebtn = localStorage.getItem("racebtn", $('.racebtn').text());
+            $('.racebtn').text(racebtn);
+        }
+        //background local storage
+        if (localStorage.getItem("backbtn") != "Backgroundarrow_drop_down") {
+            var backbtn = localStorage.getItem("backbtn", $('.backbtn').text());
+            $('.backbtn').text(backbtn);
+        }
+        //strength local storage
+        if (localStorage.getItem("strengthbtn") != "Scorearrow_drop_down") {
+            var strengthbtn = localStorage.getItem("strengthbtn", $('.strengthbtn').text());
+            $('.strengthbtn').text(strengthbtn);
+        }
+        //dexternity local storage
+        if (localStorage.getItem("DEXTERITYbtn") != "Scorearrow_drop_down") {
+            var DEXTERITYbtn = localStorage.getItem("DEXTERITYbtn", $('.DEXTERITYbtn').text());
```

Comment:
variables should generally start with lower case characters unless they are constants or classes